sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/aio/_operations/_operations.py

```diff
@@ -16,71 +15,65 @@
 from azure.core.tracing.decorator_async import distributed_trace_async
 
 from ... import models as _models
-from ...operations._operations import build_analyze_conversations_request
-
+from ..._operations._operations import build_analyze_conversation_request
 T = TypeVar('T')
+JSONType = Any
 ClsType = Optional[Callable[[PipelineResponse[HttpRequest, AsyncHttpResponse], T, Dict[str, Any]], Any]]
 
 class ConversationAnalysisClientOperationsMixin:
 
     @distributed_trace_async
-    async def analyze_conversations(
+    async def analyze_conversation(
         self,
-        conversation_analysis_options: "_models.ConversationAnalysisOptions",
-        *,
-        project_name: str,
-        deployment_name: str,
+        body: "_models.AnalyzeConversationTask",
```

Comment:
Could we use a swagger directive to rename the `body` parameter to something like `task`?